apps/browser/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@evo-ninja/core": "0.1.0",
+    "@evo-ninja/evo-agent": "0.1.0",
```

Comment:
NIT: for our package naming convention, what do you think about
```
@evo-ninja/evo-agent
@evo-ninja/js-script-writer-agent
@evo-ninja/agent-utils
```

Reply:
Convention change or did you just add "js-" prefix to the script writer agent?

Reply:
yep my bad, thought I changed more but I ended up aligning with what you already had after rewriting a few times :P